.circleci/config.yml

```diff
@@ -49,5 +57,10 @@ workflows:
           filters:
             branches:
               only: master
+      - schedule:
```

Comment:
I've never done multiple scheduled triggers in Circle before, and couldn't find anything decisive in their docs nor a quick google consultation. _Think_ this should work but if anyone knows otherwise (have to define a second workflow, can use multiple or better cron under single schedule key, etc.) please let me know!

Reply:
I think you can do it with cron!

https://crontab.guru/every-12-hours 😉

Reply:
I wasn't sure if we wanted to maintain the original schedule time but yeah that will definitely be simpler!

Reply:
You could also do `45 3,15 * * *`.